Cargo.toml

```diff
@@ -25,6 +25,7 @@ name = "svd2rust"
 repository = "https://github.com/rust-embedded/svd2rust/"
 version = "0.24.0"
 readme = "README.md"
+rust-version = "1.60"
```

Comment:
I also added [this](https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field) newly introduced attribute since we bumped the version to 1.60 which supports it.